runtime/module_test.go

```diff
@@ -204,6 +204,54 @@ func TestImportNativeModule(t *testing.T) {
 	}
 }
 
+func TestLoadMembers(t *testing.T) {
+	f := NewRootFrame()
+	var1 := NewStr("var1").ToObject()
+	var2 := NewStr("_var2").ToObject()
+	var3 := NewStr("var3").ToObject()
+	nameAttr := NewStr("__name__").ToObject()
+	allAttr := NewStr("__all__")
+	val1 := NewStr("val1").ToObject()
+	val2 := NewStr("val2").ToObject()
+	val3 := NewStr("val3").ToObject()
+	nameValue := NewStr("foo").ToObject()
+	allValue := newTestList(var1, var2).ToObject()
+
+	allDefinedDict := newTestDict(var1, val1, var2, val2, var3, val3, nameAttr, nameValue, allAttr, allValue)
+	fooAllDefinedModule := &Module{Object: Object{typ: testModuleType, dict: allDefinedDict}}
+	allUndefinedDict := newTestDict(var1, val1, var2, val2, var3, val3, nameAttr, nameValue)
+	fooAllUndefinedModule := &Module{Object: Object{typ: testModuleType, dict: allUndefinedDict}}
+
+	cases := []struct {
+		module      *Module
+		wantGlobals *Dict
+	}{
+		{
+			fooAllDefinedModule,
+			newTestDict(var1, val1, var2, val2),
+		},
+		{
+			fooAllUndefinedModule,
+			newTestDict(var1, val1, var3, val3),
```

Comment:
Would be good to test some of the corner cases like when `__all__` is not iterable.